src/main/java/appeng/parts/p2p/PartP2PFEPower.java

```diff
@@ -115,7 +140,7 @@ public int receiveEnergy( int maxReceive, boolean simulate )
 					total += received;
 				}
 
-				this.queueTunnelDrain( PowerUnits.RF, total );
+				PartP2PFEPower.this.queueTunnelDrain( PowerUnits.RF, total );
 			}
 			catch( GridAccessException ignored )
 			{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00108)